app/src/main/res/layout/fragment_watch_history.xml

```diff
@@ -128,4 +83,36 @@
 
     </androidx.coordinatorlayout.widget.CoordinatorLayout>
 
+    <LinearLayout
+        android:id="@+id/history_options"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:padding="16dp"
+        android:layout_gravity="end|bottom">
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/clear"
+            style="?attr/floatingActionButtonSmallSecondaryStyle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="end"
+            android:contentDescription="@string/clear_history"
+            android:src="@drawable/ic_delete"
+            android:tooltipText="@string/clear_history"
+            tools:targetApi="o"
+            tools:visibility="visible" />
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/play_all"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:contentDescription="@string/play_all"
+            android:src="@drawable/ic_play"
+            android:layout_marginTop="8dp"
+            android:tooltipText="@string/play_all"
+            tools:visibility="visible" />
+
+    </LinearLayout>
+
```

Comment:
In the current configuration the shadow of the buttons (especially the lower one) is cut off.

Reply:
I completely removed the rotation now.